packages/java-parser/benchmark/benchmark.js

```diff
@@ -0,0 +1,62 @@
+/* eslint no-console: 0 */
+"use strict";
+const path = require("path");
+const klawSync = require("klaw-sync");
+const _ = require("lodash");
+const fs = require("fs");
+const benchmark = require("benchmark");
+const cp = require("child_process");
+
+if (fs.existsSync(path.resolve(__dirname, "./prettier-java"))) {
+  console.log(`updating prettier-java master`);
+  cp.execSync(`git pull origin master`);
+} else {
+  console.log(`cloning prettier-java`);
+  cp.execSync(`git clone https://github.com/jhipster/prettier-java`, {
+    stdio: [0, 1, 2]
+  });
+}
+
+const masterparser = require("./prettier-java/packages/java-parser/src/index");
```

Comment:
i don't understand how would one of these paths point to the most recently release parser.

Reply:
Oh, I thought that we were comparing the master repository on github and not the npm.
I will fix it.

Reply:
It should be fixed now, it will point to the lastest parser available on npm.